File: android/src/main/java/dev/barooni/capacitor/calendar/models/inputs/GetDefaultCalendarInput.kt
package dev.barooni.capacitor.calendar.models.inputs

import com.getcapacitor.PluginCall

data class GetDefaultCalendarInput(
    private val call: PluginCall,
) {
    val useFallbackCalendar: Boolean = call.getBoolean("useFallbackCalendar", false)
}

File: ios/Plugin/Models/Inputs/GetDefaultCalendarInput.swift
import Capacitor

struct GetDefaultCalendarInput {
    private let useFallbackCalendar: Bool

    init(call: CAPPluginCall) {
        self.useFallbackCalendar = call.getBool("useFallbackCalendar", false)
    }

    func getUseFallbackCalendar() -> Bool {
        return useFallbackCalendar
    }
}
